src/editor/views/gridstackPanel.tsx

```diff
@@ -1,64 +1,159 @@
 import { Widget } from '@lumino/widgets';
 
-import { GridStack, GridStackNode, GridHTMLElement } from 'gridstack';
-
-import Cell from '../components/cell';
+import { GridStack, GridStackNode } from 'gridstack';
 
 import 'gridstack/dist/gridstack.css';
 
-export default class GridStackPanel extends Widget {
+import { GridItem, DasboardCellView } from './../components/cell';
+
+export type DasboardInfo = {
+  version: number;
+  activeView: string;
+  views: { [id: string]: DasboardView };
+};
+
+export type DasboardView = {
+  name: string;
+  type: string;
+  cellMargin: number;
+  cellHeight: number;
+  numColumns: number;
+};
+
+export class GridStackPanel extends Widget {
   constructor() {
     super();
     this.addClass('grid-panel');
-
-    this._cells = {};
+    this._cells = new Map<string, GridItem>();
   }
 
   dispose(): void {
+    console.debug('Dispose grid');
     super.dispose();
     this._cells = null;
     this._grid = null;
-    console.debug('Grid disposed:', this._grid);
+  }
+
+  get info(): DasboardInfo {
+    return this._info;
+  }
+
+  set info(info: DasboardInfo) {
+    this._info = info;
   }
 
   onUpdateRequest(): void {
     this._grid?.destroy();
 
     const grid = document.createElement('div');
     grid.className = 'grid-stack';
-    this.node.append(grid);
+    this.node.appendChild(grid);
 
     this._grid = GridStack.init({
+      animate: true,
+      removable: true,
+      removeTimeout: 500,
+      styleInHead: true,
       disableOneColumnMode: true,
-      styleInHead: true
+      resizable: { autoHide: true, handles: 'e, se, s, sw, w' }
+      //alwaysShowResizeHandle: /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
+    });
+
+    this._grid.on('change', (event: Event, items: any) => {
+      this._onChange(event, items as GridStackNode[]);
+    });
+
+    this._grid.on('removed', (event: Event, items: any) => {
+      this._onRemove(event, items as GridStackNode[]);
+    });
+
+    this._grid.on('dropped', (event: Event, items: any) => {
```

Comment:
It would be nice to keep the original types for `items` instead of `any`.

Although it's fine to stick to `any` for now if it makes development and iterating easier.